make/photon/chartserver/compile.sh

```diff
@@ -16,6 +16,12 @@ VERSION="$2"
 MAIN_GO_PATH="$3"
 BIN_NAME="$4"
 
+$ build param
```

Comment:
Pass golang cross-compilation parameters during compilation to support multi-architecture compilation